### PR TITLE
Edit JSON uplink message doc for clarity

### LINKED ISSUE #2293: Uplink message documentation comments unclear about gateway time and timestamp
#### Summary

The time and timestamp field inside the `uplink_message`=>`settings` object is non clearly explained. Especially the difference between these and the time and timestamp fields that are provided per gateway.

#### Why do we need this ?

Remove any doubt.

#### What is already there? What do you see now?

Comments that are unclear.

#### What is missing? What do you want to see?

I want to know what is the difference between the time and timestamps that are provided per gateway, versus the time and timestamp that is provided in the settings object.

#### How do you propose to document this?

Fix the comment.

#### Can you do this yourself and submit a Pull Request?

Unfortunately not because the situation is unclear to me.

See:
https://github.com/TheThingsNetwork/lorawan-stack/blame/master/doc/content/reference/application-server-data/data-formats/_index.md#L103

And compare that to:
https://github.com/TheThingsNetwork/lorawan-stack/blame/master/doc/content/reference/application-server-data/data-formats/_index.md#L120

### PR DESCRIPTION
**Pull request by benolayinka**
#### Summary

Closes #2293 

#### Changes

- ~~Clarify documentation for JSON Uplink Message timestamps.~~
- Remove time + timestamp from JSON uplink rx field

#### Testing

...

##### Regressions

...

#### Notes for Reviewers

...

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md`. The target branch is set to `master` if the changes are fully compatible with existing API, database, configuration and CLI.
- [x] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: doc/content/integrations/data-formats/_index.md [hunk 1]
```diff
@@ -116,8 +116,8 @@ The JSON uplink messages use the following format:
       "data_rate_index": 5,                  // LoRaWAN data rate index
       "coding_rate": "4/6",                  // LoRa coding rate
       "frequency": "868300000",              // Frequency (Hz)
-      "timestamp": 2463457000,               // Timestamp of the gateway concentrator when the message has been received
-      "time": "2020-02-12T15:15:45.787Z"     // ISO 8601 UTC timestamp at which the uplink has been received by the gateway
+      "timestamp": 2463457000,               // Timestamp of the gateway concentrator when the accepted uplink message has been received
+      "time": "2020-02-12T15:15:45.787Z"     // ISO 8601 UTC timestamp at which the accepted uplink has been received by the gateway
```
Comment (johanstokking) [inline doc/content/integrations/data-formats/_index.md:120]: Actually I think we should reset these fields here so they disappear, @rvolosatovs what do you think?

If not, it's not really "accepted" but "deduplicated", and I'm afraid it's pretty random which one is chosen here

Comment (benolayinka) [reply doc/content/integrations/data-formats/_index.md:120]: Reset them where?

Comment (johanstokking) [reply doc/content/integrations/data-formats/_index.md:120]: In the Network Server

Comment (rvolosatovs) [reply doc/content/integrations/data-formats/_index.md:120]: Network Server discards these, so these are only seen and used by Gateway Server.

Comment (johanstokking) [reply doc/content/integrations/data-formats/_index.md:120]: > Network Server does not use these, so they are only used by Gateway Server.

Gateway Server sets them as part of the individual `RxMetadata`. It remains useful in there.

> I think NS should unset them as well, in fact.

That's what I mean. It doesn't make sense anymore to have this information _after_ deduplication in the `Settings`; but they should stay in the settings

Comment (johanstokking) [reply doc/content/integrations/data-formats/_index.md:120]: @benolayinka remove these two fields here for now as we'll get rid of this

Please also file issue for unsetting these field values in NS after deduplication

**Comment by benolayinka** [comment]
> Is the issue filed for unsetting these fields in NS?
> 
> I don't see the (back)reference here

Filed it but I'm not sure what it means to unset, so the issue could use improvement

**Pull request merged**